prisma/schema.prisma

```diff
@@ -28,25 +28,37 @@ model Event {
 }
 
 model User {
-  id                  Int       @id @default(autoincrement())
-  created_at          DateTime  @default(now()) @db.Timestamp(6)
-  updated_at          DateTime  @default(now()) @updatedAt @db.Timestamp(6)
-  email               String    @db.VarChar @unique(map: "uq_users_on_email")
-  graffiti            String    @db.VarChar @unique(map: "uq_users_on_graffiti")
-  total_points        Int       @default(0)
-  country_code        String    @db.VarChar
-  email_notifications Boolean   @default(false)
-  last_login_at       DateTime? @db.Timestamp(6)
-  discord             String?   @db.VarChar
-  telegram            String?   @db.VarChar
-  github              String?   @db.VarChar
+  id                  Int         @id @default(autoincrement())
+  created_at          DateTime    @default(now()) @db.Timestamp(6)
+  updated_at          DateTime    @default(now()) @updatedAt @db.Timestamp(6)
+  email               String      @db.VarChar @unique(map: "uq_users_on_email")
+  graffiti            String      @db.VarChar @unique(map: "uq_users_on_graffiti")
+  total_points        Int         @default(0)
+  country_code        String      @db.VarChar
+  email_notifications Boolean     @default(false)
+  last_login_at       DateTime?   @db.Timestamp(6)
+  discord             String?     @db.VarChar
+  telegram            String?     @db.VarChar
+  github              String?     @db.VarChar
   events              Event[]
+  node_uptime         NodeUptime?
 
   @@index([email], name: "index_users_on_email")
   @@index([graffiti], name: "index_users_on_graffiti")
   @@map("users")
 }
 
+model NodeUptime {
+  user            User     @relation(fields: [user_id], references: [id])
```

Comment:
nit: I prefer to have separate `id` / independent primary keys unless it's a join table. I've found it to be cleaner and easier to manage, but either way